electrical/RPi BreakBeam.py

```diff
@@ -0,0 +1,24 @@
+import RPi.GPIO as GPIO
+
+BEAM_PIN = 17
+beam_broken = False
+
+def break_beam_callback(channel):
+    if GPIO.input(BEAM_PIN):
+        beam_broken = False
+        print(beam_broken)

```

Comment:
```suggestion
        # print(beam_broken)

```
Maybe we can comment these out and uncomment them when testing?